frontend/src/scenes/sceneLogic.tsx

```diff
@@ -87,6 +89,15 @@ const pathPrefixesOnboardingNotRequiredFor = [
     urls.oauthAuthorize(),
 ]
 
+const DelayedLoadingSpinner = (): JSX.Element => {
+    const [show, setShow] = useState(false)
+    useEffect(() => {
+        const timeout = window.setTimeout(() => setShow(true), 500)
```

Comment:
could lower this a bit to 250 but not sure the right number to make it feel faster